seeders/20200206005427-demo-patient.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const db = require("../models");
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const user = await db.user.create({
+      firstName: "Darth",
+      lastName: "Vader",
+      email: "darthvader@sithlords.com",
+      password: "ga1axyru1er",
+      createdAt: new Date(),
+      updatedAt: new Date()
+    });
+
+    const event = await db.event.create({
+      eventDate: new Date(),
+      name: "Death Star Launch Day",
+      createdBy: user.id,
+      isActive: true,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    });
+
+    const collectionPoint = await db.collectionPoint.create({
+      name: "Checkpoint Tatooine",
+      eventID: event.id,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    });
+
+    return queryInterface.bulkInsert('patients', [{
+      gender: 'Male',
+      age: 19,
+      barcodeValue: 1525242,
+      collectionPointId: collectionPoint.id,
```

Comment:
`collectionPointId` -> `collectionPoint`
```suggestion
      collectionPoint: collectionPoint.id,
```